refinery/ui/source/js/file-browser/ctrls/ctrl.spec.js

```diff
@@ -47,6 +51,25 @@
       expect(ctrl.collapsedToolPanel).toEqual(true);
     });
 
+    describe('Toggle Edit Mode', function () {
+      it('toggleEditMode is method', function () {
+        expect(angular.isFunction(ctrl.toggleEditMode)).toBe(true);
+      });
+      it('toggleEditMode updates editMode', function () {
+        ctrl.toggleEditMode();
+        expect(ctrl.editMode).toBe(true);
+      });
```

Comment:
```suggestion
        ctrl.toggleEditMode();
        expect(ctrl.editMode).toBe(false);
      });
```

And to be sure, toggle it back? (Also, trying out a new github feature.)